tests/sls/homu/toml.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+import toml
+
+from tests.util import Failure, Success
+
+
+def run():
+    config_path = os.path.join('/home', 'servo', 'homu', 'cfg.toml')
+    with open(config_path) as conf:
+        try:
+            toml.loads(conf.read())
```

Comment:
We can use `toml.load` instead which will handle opening and closing the file for us.